examples/sumo/grid.py

```diff
@@ -4,9 +4,114 @@
 from flow.core.params import SumoParams, EnvParams, InitialConfig, NetParams
 from flow.core.params import VehicleParams
 from flow.core.params import TrafficLightParams
+from flow.core.params import InFlows
 from flow.envs.loop.loop_accel import AccelEnv, ADDITIONAL_ENV_PARAMS
 from flow.scenarios.grid import SimpleGridScenario
 
+# set to true if you would like to run the experiment with inflows of vehicles
+# from the edges, and false otherwise
+USE_INFLOWS = False
```

Comment:
The simulation breaks upon setting `USE_INFLOWS = True`. Can you look into this?

Reply:
I think the issue is that the list of edge names generated by the function gen_edges() is not the same as the list of edges in the network. I get the following error in SUMO:

Error: The route 'routeleft3_0' for flow 'flow_0' is not known.

Here is the list of edges by gen_edges() (on 2x3 grid):

['left3_0', 'right0_0', 'left3_1', 'right0_1', 'bot0_0', 'top0_2', 'bot1_0', 'top1_2', 'bot2_0', 'top2_2']

and the edges output by env.k.scenario.get_edge_list():

['bot0_1', 'top0_1', 'top1_2', 'bot0_2', 'right2_0', 'right1_0', 'bot1_0', 'bot1_3', 'top1_0', 'top1_3', 'right0_0', 'left1_2', 'right0_1', 'right0_2', 'top1_1', 'bot1_1', 'left1_0', 'right2_1', 'left2_1', 'right1_1', 'bot0_0', 'top0_2', 'right2_2', 'left0_1', 'top0_3', 'left0_2', 'bot1_2', 'left2_2', 'right1_2', 'left1_1', 'left2_0', 'bot0_3', 'top0_0', 'left0_0']

Reply:
I think this might be the fix:

change

        for i in range(col_num):
            edges += ["left" + str(row_num) + '_' + str(i)]
            edges += ["right" + '0' + '_' + str(i)]

        # build the left and then the right edges
        for i in range(row_num):
            edges += ["bot" + str(i) + '_' + '0']
            edges += ["top" + str(i) + '_' + str(col_num)]

to

        for i in range(col_num):
            edges += ["left" + str(row_num-1) + '_' + str(i)]
            edges += ["right" + '0' + '_' + str(i)]

        # build the left and then the right edges
        for i in range(row_num):
            edges += ["bot" + str(i) + '_' + '0']
            edges += ["top" + str(i) + '_' + str(col_num-1)]

However, I am not sure. Since when I changed this in all the files, one test in test_green_wave.py failed:

`======================================================================
FAIL: test_get_distance_to_intersection (__main__.TestUtils)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/ashkany/Documents/GitHub/flow/tests/fast_tests/test_green_wave.py", line 87, in test_get_distance_to_intersection
    self.assertListEqual(d_list, dists[0])
AssertionError: Lists differ: [94.0, 84.0, 74.0, 64.0, 54.0] != []`

Reply:
found the bug. I have row_num and col_num flipped when I called the `gen_edges` method. It should be good now

Reply:
Yes this works! Thanks

Reply:
@kjang96 are you happy with the fix? If so, i will merge it